.travis.yml

```diff
@@ -2,6 +2,7 @@ language: ruby
 cache: bundler
 bundler_args: --without development
 script:
+ - RAILS_ENV=test bundle exec rake bundler:audit
```

Comment:
as with https://github.com/alphagov/trade-tariff-admin/pull/43 needs to go in a new PR after updates